Sources/Public/Configuration/LottieConfiguration.swift

```diff
@@ -44,4 +46,11 @@ public struct LottieConfiguration: Hashable {
   /// The color space to be used for rendering
   ///  - Defaults to `CGColorSpaceCreateDeviceRGB()`
   public var colorSpace: CGColorSpace
+
+  /// Whether or not transform and position changes of the view should animate alongside
+  /// any existing animation context.
+  ///  - Defaults to `true` which will grab the current animation context and animate position and
+  ///    transform changes matching the current context's curve and duration.
+  ///    `false` will cause transform and position changes to happen unanimated
+  public var animatesLayoutWithinCurrentContext: Bool
```

Comment:
Can this be an instance property on `LottieAnimationView` instead of a property in `LottieConfiguration`?

Reply:
`LottieConfiguration` is mostly for properties that we need to be able to configure globally for the whole app. I imagine in this case we'd rather configure this flag for the specific `LottieAnimationView` that is encountering this bug.